hdmi_cec_driver.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2024 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+#include <string.h>
+#include <stdlib.h>
+#include "ccec/drivers/hdmi_cec_driver.h"
+
+HDMI_CEC_STATUS HdmiCecOpen(int* handle)
+{
+  if (handle == NULL)
+  {
+	  return HDMI_CEC_IO_INVALID_ARGUMENT;
+  }
+  return HDMI_CEC_IO_SUCCESS;
+}
+
+HDMI_CEC_STATUS HdmiCecClose(int handle)
+{
+  if (handle == 0)
+  {
+    return HDMI_CEC_IO_INVALID_ARGUMENT;
+  }
+  return HDMI_CEC_IO_SUCCESS;
+}
+
+HDMI_CEC_STATUS HdmiCecGetPhysicalAddress(int handle, unsigned int* physicalAddress)
+{
+  if (handle == 0 || physicalAddress == NULL)
+  {
+    return HDMI_CEC_IO_INVALID_ARGUMENT;
+  }
+  return HDMI_CEC_IO_SUCCESS;
+}
+
+HDMI_CEC_STATUS HdmiCecAddLogicalAddress(int handle, int logicalAddresses)
+{
+  if (handle == 0 || logicalAddresses < 0x0 || logicalAddresses > 0xF )
```

Comment:
Logical address is of size int. Please make sure that the boundary values are correct.